extension/index.googleFunction.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * If you are deploying extension module as a serverless function,
+ * this will be the main javascript file.
+ *
+ * This function has tested as Google Cloud Function
```

Comment:
```suggestion
```
does not make sense.

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/528/commits/615b5839a69ff194b3cbf1e2e70226b027606f28

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/528/commits/476f9545083895617aaa53af788a20e932b90983